doc/start/os-recommendations.rst

```diff
@@ -39,6 +39,31 @@ platforms.  Generally speaking, there is very little dependence on
 specific distributions aside from the kernel and system initialization
 package (i.e., sysvinit, upstart, systemd).
 
+Nautilus (14.2.z)
```

Comment:
Would it make sense to clean up the tables for EOL releases from this page as well?

Reply:
Sure I think that would make sense